synthtool/gcp/templates/python_library/README.rst

```diff
@@ -0,0 +1,90 @@
+Python Client for {{ metadata['repo']['name_pretty'] }} API
```

Comment:
General annoying question.  Why is this a `rst` instead of `md` file? Is there a good reason for it to be different from every other language?

Reply:
No particular reason.  The doc generator. can pull in MD's as well. Will modify. 👍